MeesGame/AI/IAI.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace AI
+{
+    /// <summary>
+    /// This interface defines all functions the AI player will ever call.
+    /// Any AI should implement these and should not depend on any other function calls.
+    /// This should make it easy to connect the AI to the game and make sure the AI won't do anything else.
+    /// The AI is allowed to make dummy players and move them around.
+    /// </summary>
+    interface IAI
+    {
+        /// <summary>
+        /// Only called at game start.
+        /// </summary>
+        /// <param name="player">The player the AI will take control of.</param>
+        void GameStart(MeesGame.TimedPlayer player);
```

Comment:
Can we make the parameter to this function an interface (IAIPlayer or something?